catalog/spatial/registry/registry-federation-admin-service/src/main/java/org/codice/ddf/registry/federationadmin/service/FederationAdminService.java

```diff
@@ -180,6 +180,22 @@ void updateRegistryEntry(Metacard metacard, Set<String> destinations)
     List<Metacard> getRegistryMetacards() throws FederationAdminException;
 
     /**
+     * Get a list of internal registry metacards.
+     *
+     * @return List<Metacard>
+     * @throws FederationAdminException If any exception was thrown by the call to CatalogFramework.query()
```

Comment:
Since this is an interface, seems like you should not have a mention in here about the `CatalogFramework.query()` but maybe you have it here for good reason.